srcpkgs/howl/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'howl'
+pkgname=howl
+version=0.6
+revision=1
+archs="i686 x86_64 x86_64-musl"
+build_wrksrc=src
```

Comment:
I think that this should not be explicitly need to be specified.

Reply:
Yes, it needs to change to `src/` to build.

Reply:
It is part of the original instructions in the [GitHub readme](https://github.com/howl-editor/howl). Without it, it fails:
```
...
=> howl-0.6_1: running do_build ...
make: *** No targets specified and no makefile found.  Stop.
=> ERROR: howl-0.6_1: do_build: '${make_cmd} ${makejobs} ${make_build_args} ${make_build_target}' exited with 2
=> ERROR:   in do_build() at common/build-style/gnu-makefile.sh:14
```